ferrocene/doc/user-manual/src/rustfmt/cli.rst

```diff
@@ -38,7 +52,7 @@
 
       Short option: ``-l``.
 
-      Prints the names of mismatched files that were formatted. Prints the names of files that would be formatted when used with ``--check`` mode.
+      Prints the names of mismatched files that were formatted. Prints the names of files that would be formatted when used with ``--check``.
```

Comment:
Would or should?
```suggestion
      Prints the names of mismatched files that were formatted. Prints the names of files that should be formatted when used with ``--check``.
```

Reply:
_would_ fits in that these files would be formatted if the `--check` flag was not used

Reply:
So then we say "that would be formatted when not used with `--check`"?

Reply:
hm, this is actually complex

how about 2e97dfee919

Reply:
Mhh, I still don't think it is correct, but maybe it is also me not being a native speaker and therefore understanding it differently.

What about?
>  When used with --check, this option instead prints the names of files that are not well-formatted.

Reply:
that is totally better... 54ae328fc6b